src/components/app.jsx

```diff
@@ -47,6 +47,12 @@ const App = React.createClass({
               </div>
             </section>
             {this.props.children}
+            <footer style={{marginTop: "5em"}}>
+              <hr/>
+              <a href={config.gitWebpageUrl} rel="external" target="_blank">
```

Comment:
I did not find references to that, and had never seen it. Would you have a source for it to be a good practice? 😃